boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java

```diff
@@ -86,7 +86,7 @@ private void createSpringBootDockerFile(String springBootVersion) throws MojoExe
     }
     
     @SuppressWarnings("resource")
-    private boolean isFileExecutable(File file) throws IOException {
```

Comment:
Can we just have it throw a `MojoExecutionException`? I would catch the `IOException` and throw a `MojoExecutionException` in that case, passing the `IOException` through with an error message relevant to this method.

Reply:
DockerBuildMojo.execute() is catching Exception and throwing a MojoExecutionException